src/Components/Navbar/index.stories.js

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { ButtonIcon } from 'react-rainbow-components/components';
-import { faSlidersH, faChalkboardTeacher} from '@fortawesome/free-solid-svg-icons';
-import { faBell} from '@fortawesome/free-regular-svg-icons';
-import { withKnobs, number } from '@storybook/addon-knobs';
-import NavbarStyle1 from './Style1';
-import NavbarStyle2 from './Style2';
+import React from './node_modules/react';
+import { storiesOf } from './node_modules/@storybook/react';
+import { FontAwesomeIcon } from './node_modules/@fortawesome/react-fontawesome'
+import { ButtonIcon } from './node_modules/react-rainbow-components/components';
+import { faSlidersH, faChalkboardTeacher} from './node_modules/@fortawesome/free-solid-svg-icons';
+import { faBell} from './node_modules/@fortawesome/free-regular-svg-icons';
+import { withKnobs, number } from './node_modules/@storybook/addon-knobs';
+import NavbarStyle1 from './style1';
+import NavbarStyle2 from './style2';
```

Comment:
Change import names to be with uppercase `S` or change folders to be lowercase `s`.